doc/developer-guides/hld/hld-devicemodel.rst

```diff
@@ -473,7 +475,7 @@ The following table summarizes the emulation of accesses to I/O port
 I/O Client Interfaces
 =====================
 
-.. note:: replace with reference to API documentation
+.. note:: Replace with reference to API documentation.
```

Comment:
Is this an internal note, saying to replace this section with API doc?

Reply:
This should reference our API documentation.  I can find these functions in the 2.0 docs (https://projectacrn.github.io/2.0/api/GVT-g_api.html) but they're gone in the current release (I think GVT-D was remove in v2.6).  @NanlinXie @gvancuts can you help here: should this section be deleted now?

Reply:
If I delete this section, there are 2 references to it in this file at lines 396 and 702. Please advise on how to edit those lines.

Reply:
I don't know to be honest, @NanlinXie , can you get someone from the engineering team to clarify what we should do with these APIs. They did not _seem_ GVT-g specific but it's the only place we could find them and they seem gone. Have they been replaced by other APIs?

Reply:
For I/O client registration

acrn_ioreq_create_client - create ioreq client

acrn_ioreq_add_iorange - add iorange monitored by ioreq client

acrn_ioreq_intercept_bdf - set intercept bdf info of ioreq client

acrn_ioreq_get_reqbuf - get request buffer

I/O client runtime helpers.

acrn_ioreq_attach_client - start handle request for ioreq client

acrn_ioreq_complete_request - notify guest request handling is completed

For I/O client destruction

acrn_ioreq_destroy_client - destroy ioreq client

acrn_ioreq_del_iorange - del iorange monitored by ioreq client

acrn_ioreq_unintercept_bdf - clear intercept bdf info of ioreq client

they are device model API , not GVT-g API , need to add it to device model API page

Reply:
> they are device model API , not GVT-g API , need to add it to device model API page

They are not. None of these exist in the current codebase. What have they been replaced by?

There are a few that _resemble_ those here: https://github.com/projectacrn/acrn-hypervisor/blob/master/devicemodel/include/vmmapi.h#L98-L102. Is that it?

Reply:
@liuhang-bit Please double-check existence of parameters. Cannot find them in code in repo. See Geoffroy's comment.

Reply:
yes , the latest API is just like in the link above, so we should remove the obsolete one

Reply:
> 
Can you provide for each of the APIs we need to include a brief description? I don't think Amy has yet quite enough info for this section.

Reply:
@liuhang-bit Please add Doxygen comments to vmmapi.h, so we can pull the comments into this hld.

Reply:
Per offline discussion with @dbkinder @NanlinXie @liuhang-bit, we will resolve this issue soon after the 2.7 release. Issue: https://github.com/projectacrn/acrn-hypervisor/issues/6918. For now, please merge as is for 2.7.